src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/measure/ActionMeterTest.kt

```diff
@@ -158,6 +160,37 @@ class ActionMeterTest {
         )
     }
 
+    @Test
+    fun shouldHookAction() {
+        val output = CollectionActionMetricOutput(mutableListOf())
+        val countingPostMeasurmentAction = CountingPostMeasurmentAction()
+        val actionMeter = ActionMeter.Builder(
+            output = output
+        )
+            .virtualUser(vu)
+            .clock(Clock.fixed(start, ZoneId.of("UTC")))
+            .postMeasurementAction(countingPostMeasurmentAction)
+            .build()
+
+        actionMeter.measure(CREATE_ISSUE) { }
+        actionMeter.measure(VIEW_BOARD) { }
+
+        assertThat(countingPostMeasurmentAction.getCount()).isEqualTo(2)
+
+    }
+
+    private class CountingPostMeasurmentAction : PostMeasurementAction {
```

Comment:
`Action` already has a connotation in `jira-actions`. I'd call it directly: `PostMeasurementHook` or `PostMetricHook`.